### PR TITLE
protoc-gen-openapiv2: Fix so that FieldMasks descriptions are correctly generated

### LINKED ISSUE #1556: protoc-gen-swagger: Incorrect description generated for google.protobuf.FieldMask
## 🐛 Bug Report

When generating a Swagger definition, the `description` for fields of type `google.protobuf.FieldMask` is not taken from the field's comment, as it is for other types, but is always `The set of field mask paths.`.

## To Reproduce

Create a `proto` file with a service that takes a message containing a `google.protobuf.FieldMask`:

```
syntax = "proto3";

package test;

import "google/api/annotations.proto";
import "google/protobuf/empty.proto";
import "google/protobuf/field_mask.proto";

service Foo {
    rpc Bar(Request) returns (google.protobuf.Empty) {
        option (google.api.http) = {
            get: "/foo"
        };
    }
}

message Request {
    // This should be generated as a description.
    google.protobuf.FieldMask a_mask = 1;
}
```

Generate the Swagger definition using `protoc -I. --swagger_out=...`

## Expected behavior

The `description` for the FieldMask parameter should be `This should be generated as a description.`.

## Actual Behavior

The `description` for the FieldMask parameter is `The set of field mask paths.`.

## Your Environment

go version go1.14.4 darwin/amd64
github.com/grpc-ecosystem/grpc-gateway v1.14.6

### PR DESCRIPTION
**Pull request by krosen040**
#### References to other Issues or PRs

Fixes #1556

#### Have you read the [Contributing Guidelines](https://github.com/grpc-ecosystem/grpc-gateway/blob/master/CONTRIBUTING.md)?

Yes

#### Brief description of what is fixed or changed

Added `google.protobuf.FieldMask` as a well-known-type in `protoc-gen-openapiv2` package so that fields of this type are generated with the correct description in Swagger definitions.

#### Other comments

This also had some other side effects, which I'm not sure are considered breaking changes.

* The name of the `FieldMask` fields in the Swagger definition will no longer be suffixed with `.paths`
* The `protobufFieldMask` type definition will no longer be added to the Swagger definition

### CODE REVIEW

**Comment by krosen040** [comment]
@googlebot I signed it!

**Comment by krosen040** [comment]
@googlebot I fixed it.

File: protoc-gen-openapiv2/internal/genopenapi/template.go [hunk 1]
```diff
@@ -25,6 +25,12 @@ import (
 )
 
 var wktSchemas = map[string]schemaCore{
+	".google.protobuf.FieldMask": {
+		Type: "array",
+		Items: (*openapiItemsObject)(&schemaCore{
+			Type: "string",
+		}),
+	},
```
Comment (johanbrandhorst) [inline protoc-gen-openapiv2/internal/genopenapi/template.go:33]: Could you add a comment here stating that this must match up with the behaviour of our JSON unmarshaler, e.g. https://github.com/protocolbuffers/protobuf-go/blob/v1.25.0/encoding/protojson/well_known_types.go#L860-L885. It looks to me like it interprets a comma-separated slice of paths, so this description looks right.

Comment (krosen040) [reply protoc-gen-openapiv2/internal/genopenapi/template.go:36]: I added a comment

**Comment by johanbrandhorst** [comment]
Thanks for your contribution!

**Pull request merged**